components/molecules/BenefitApplicationCard.js

```diff
@@ -26,42 +28,36 @@ const BenefitApplicationCard = (props) => {
   }
 
   return (
-    <div className="benefit-card px-4 md:px-6 py-7" data-cy="adcard">
-      <div className="mx-auto sm:grid sm:grid-cols-4 sm:divide-x-2">
+    <div className="benefit-card" data-cy="adcard">
+      <div className="flex flex-col pt-6">
         <div
           id={`${benefitType}-${benefitSubType}-app-card`}
-          className="col-span-1 py-4"
+          className="col-span-1 py-4  px-4 sm:px-16 pt-6"
         >
-          <h2 className="font-bold font-display text-4xl sm:text-lg md:text-xl lg:text-3xl xl:text-4xl">
+          <h2 className="font-bold font-display text-4xl sm:text-6xl">
             {benefitName}
           </h2>
-          <div className="font-bold font-display text-md sm:text-md lg:text-md mb-2 w-44 sm:w-32 lg:w-44">
+          <div className="font-bold font-display text-md sm:text-md lg:text-md mb-2 w-44 sm:w-32 lg:w-44 ">
             {t[benefitSubType]}
           </div>
+          <ul className="pb-6 sm:mb-7">
```

Comment:
I don't think we need a list here. Leaving just the `<p>` should be fine. 

This might get confusing to the screen readers.

Reply:
I copied the the universal card for consistency, i am fine either way.

Reply:
I'm not sure just the p tags is best, there should be some kind of association between the two texts. The status acting like a "label".

Just some context: It was originally suggested we use a data-definition-list for the summaries but because of the links in some, we opted for a regular unordered list; that patter is copied over here.

Reply:
OK, so we should leave it a list then?